contrib/utilities/indent

```diff
@@ -61,11 +71,74 @@ fi
 echo "--- Found astyle 2.04 at ${astyle}"
 
 
-# collect all header and source files and process them in batches of 50 files
-# with up to 10 in parallel
+# Indent a file.
+#
+# We use astyle 2.04 for indentation, which is a tool that predates C++11
+# and that consequently has trouble with things such as
+#   std::vector<Point<dim>>
+# where it interprets the closing `>>` as a right shift operator. It then
+# gets its template closing angle bracket counter all out of whack and
+# produces awkward indentations. To avoid this, we first convert all closing
+# `>>` into `> >`, run astyle, and then change it back. This leads to reasonable
+# results.
+#
```

Comment:
Is this documentation still up-to-date after #4614 and #4635?

Reply:
Yes. The way I interpret the flag used for #4614 and made the default in #4635 is that it "keeps the template brackets close to the text", not that it deals with the closing brackets as ">>".